pkg/security/security_profile/activity_tree/activity_tree.go

```diff
@@ -347,6 +359,13 @@ func GetNextAncestorBinaryOrArgv0(entry *model.ProcessContext) *model.ProcessCac
 	return nil
 }
 
+func eventHaveValidCookie(entry *model.ProcessCacheEntry) bool {
+	if !entry.ExecTime.IsZero() && entry.Cookie != 0 {
```

Comment:
nit: you could just do return `!entry.ExecTime.IsZero() && entry.Cookie != 0` ?

Reply:
Suggestion applied, thanks